src/nickel.rs

```diff
@@ -347,25 +346,28 @@ mod tests {
     use std::str::FromStr;
     use std::net::SocketAddr;
 
-    #[test]
+    #[tokio::test]
     #[should_panic(expected = "invalid socket address")]
```

Comment:
Tests are fine but this is testing that `ToSocketAddrs` fails, which is part of the standard library.  Not a blocker but still a weird test to have.